### DISCUSSION TITLE
Specific organ permeability formula

### DISCUSSION THREAD

**Discussion by Vincent-AC** [Open-Systems-Pharmacology/Forum#1165] [Questions & Problems] [answered]
Hello,

I am trying to find the source for the formula used to compute the specific organ permeability parameter which is the following : 
`IsSmallMolecule ? (MWEff *1E9 / 336) ^ (-6) * 10^LogMA / 5 * 1E-4 *1E-1: 0`

Thanks in advance,

**Comment by AndreDlm** [comment]
Hi @Vincent-AC, I think the source of this formula is not well documented, but if I'm not wrong it was developed using ciprofloxacin as reference compound; therefore, you can find its molecular weight and lipophilicity in the denominator of the equation. 

More generally, lipophilicity (`logMA`) as input variable can be deduced from Fick's law of diffusion (by expressing the concentration in the membrane in terms of the free concentration and partitioning into the membrane with `logMA` as partition coefficient) and the effective molecular weight `MWEff` is a surrogate for molecular volume which also affects the diffusion coefficient according to the Einstein-Stokes equation. I would have to check where the exponent `^(-6)` comes from, but I believe it from this book chapter [here](https://www.sciencedirect.com/science/article/abs/pii/S0070216108603408).

And for large molecules, passive diffusion across cell membranes was neglected as described in this [paper](https://link.springer.com/article/10.1007/s10928-017-9559-4).

**Reply by Vincent-AC** [reply]
Hi @AndreDlm Thank you for your answer. The paper about the ^-6 exponent is quite interesting indeed. 
However, I still don't understand the equation given that it is a bit different from the one in the cited paper.
Also I don't really get your point about ciprofloxacin as I cannot find a reference for a MW of 336 or a P of 5 (logP ~ 0.70) e.g. this [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6267540/)

**Reply by AndreDlm** [reply]
Hi @Vincent-AC, note that the value of 336 is *not* the molecular weight, but the effective molecular weight (`MWEff`). The source of the calculation method for `MWEff` is referenced in this [paper](https://www.sciencedirect.com/science/article/pii/S0022354915318244) and described in greater detail in this [paper](https://pubs.acs.org/doi/abs/10.1021/jm030999b). I don't know, however, from which reference this value was taken from. The lipophilicity of 5 in the denominator is the absolute value (not log-transformed, so 10^0.7 = 5.01). 
Concerning the exact derivation of the equation I have to admit that I would have to dig a bit deeper in it. Maybe someone from the community can help?

**Comment by wschm** [comment, marked as answer]
Dear @Vincent-AC , Andre's answer was fully correct. Actually I developed the equation many years ago, but never found the time to publish the approach. It is simply based on the assumption that permeability of cell membranes is propertional to the partition coefficient of the lipid core of these membranes, i.e. P = 10^logP and proportional to MW to the power of six. The large power of six is the upper end of the range of experimentally determined values you may find in the literatur. However, it turned out, that for many different compounds I worked with, this power led to the best results as compared to smaller values. The rest of the approach is then simply a recalculation of permeability values determined from fitting to detailed PK-data for Ciprofloxacin according to the rule of proportion. Admittedly the equation is based on only limited data and certainly lacks a stringent statistics based derivation. However, it worked extremely well in many projects, why we decided not to detain it from the community, when developing PK-Sim.

**Comment by Vincent-AC** [comment]
Dear @AndreDlm and @wschm 
Thank you very much for your answers I do understand the formula now ! 
I mark Walter's comment as answer but I would tag all your answers if I could.
Best regards,
Vincent

**Comment by PavelBal** [comment]
@Yuri05 Maybe this should go to the documentation?
